public/usage-examples/geometry/cicle_intersects_triangle-1-example-opp.cs

```diff
@@ -0,0 +1,46 @@
+using SplashKitSDK;
+
+public class Program
+{
+    public static void Main()
+    {
+        // A fixed triangle is placed as the target.
+        Window gameWindow = new Window("Circle-Triangle Intersect", 600, 600);
+        Triangle target = SplashKit.TriangleFrom(300, 100, 250, 300, 350, 300);
+
+        // A circle moves with arrow keys.
+        Circle playerCircle = new Circle
+        {
+            Radius = 20,
+            Center = SplashKit.PointAt(100, 100)
+        };
+
+        while (!gameWindow.CloseRequested)
+        {
+            SplashKit.ProcessEvents();
+
+            if (SplashKit.KeyDown(KeyCode.UpKey)) playerCircle.Center.Y -= 2;
```

Comment:
The Thoth Tech style guide says that all 'if' statements need to be on their own line. Do this for all four key input lines:
```suggestion
            if (SplashKit.KeyDown(KeyCode.UpKey))
                {
                    circleShape.Center.Y -= 2;
                }
```